torax/_src/transport_model/tglf_based_transport_model.py

```diff
@@ -0,0 +1,285 @@
+    #   psi/2π
+    #   https://gafusion.github.io/doc/geometry.html#effective-field
+    #   https://gafusion.github.io/doc/cgyro.html#faq
+    # - c_s (ion sound speed)
+    #   https://gafusion.github.io/doc/cgyro/outputs.html#output-normalization
+    m_D_amu = 2.014  # Mass of deuterium [amu] TODO: load from lookup table
+    m_D = m_D_amu * constants.mp  # Mass of deuterium [kg]
+    c_s = (T_e / m_D) ** 0.5  # T_e [J], m_D [kg], gives c_s in [m/s]
+    a = geo.a_minor  # Device minor radius at LCFS [m]
+    r = geo.r_mid_face  # Flux surface centroid minor radius [m]
+    B_unit = (
+        core_profiles.q_face
+        / (2 * jnp.pi * r)  # Note: psi_TGLF is psi_TORAX/2π
+        * jnp.gradient(core_profiles.psi.face_value(), r)
+    )
+
+    # Temperature ratio
+    Ti_over_Te = core_profiles.T_i.face_value() / core_profiles.T_e.face_value()
+
+    # Dimensionless gradients
+    normalized_log_gradients = quasilinear_transport_model.NormalizedLogarithmicGradients.from_profiles(
+        core_profiles=core_profiles,
+        radial_coordinate=geo.r_mid,  # On the cell grid
+        reference_length=a,
+    )
+
+    # Dimensionless electron-electron collision frequency = nu_ee / (c_s/a)
+    # https://gafusion.github.io/doc/tglf/tglf_list.html#xnue
+    # https://gafusion.github.io/doc/cgyro/cgyro_list.html#cgyro-nu-ee
+    # - In the TGLF docs, XNUE is labelled as electron-ion collision
+    #   frequency. It is believed that it should actually be the electron-
+    #   electroncollision frequency, see
+    #   https://pyrokinetics.readthedocs.io/en/latest/user_guide/collisions.html#tglf
+    # - In the TGLF docs, nu_ee is shown in CGS units. Below is the SI version.
+    # - TGLF uses a slightly different calculation of log_Lambda to those in
+    #   Wesson, namely
+    #       log_Lambda = (
+    #         24.0 - 0.5 * jnp.log(n_e_1e19 * 1e13) + jnp.log(T_e_keV * 1e3)
+    #       )
+    #   https://github.com/gafusion/gacode/blob/740bb2dc811bc1ad38be65ea4b87330995931305/cgyro/src/cgyro_make_profiles.F90#L145
+    #   This is different to Wesson by about ~0.5%. Below, we use the TLGF
+    #   version, but with n_e [m^-3] and T_e [J].
+    log_Lambda = 74.2 - 0.5 * jnp.log(n_e) + jnp.log(T_e)
+    nu_ee = (jnp.sqrt(2) * n_e * constants.qe**4 * log_Lambda) / (
+        16 * jnp.pi * constants.epsilon0**2 * constants.me**0.5 * T_e**1.5
+    )
+    normalized_nu_ee = nu_ee / (c_s / a)
+
+    # Dimensionless safety factor shear
+    # https://gafusion.github.io/doc/tglf/tglf_list.html#tglf-q-prime-loc
+    # - TGLF definition is q^2 a^2 s / r^2
+    #   where s = r/q dq/dr
+    #   (https://gafusion.github.io/doc/cgyro/cgyro_list.html#cgyro-s)
+    q_prime = (
+        psi_calculations.calc_s_face(geo, core_profiles.psi)
+        * core_profiles.q_face**2
+        * a**2
+        / r**2
+    )
+
+    # Electron beta
+    # https://gafusion.github.io/doc/tglf/tglf_list.html#tglf-betae
+    # https://gafusion.github.io/doc/cgyro.html#faq
```

Comment:
the cgyro faq are also referred to above, could we have a small section on documentation that also has the main links in it for this model?